src/dealDashboard/dealVotes/dealVotes.html

```diff
@@ -5,35 +5,47 @@
     <div class="content">
       <h1 class="title heading heading3 gradient">Vote</h1>
       <p class="bodySmallText">
-        You’re eligible to vote. If you choose not to vote, you will automatically reject the deal.
+        ${statusText.voteText}
       </p>
-      <pbutton-group>
-        <pbutton class="acceptButtonActive" full-width type="formfield">
+      <pbutton-group if.bind="ethereumService.defaultAccountAddress && deal.representativeDAO(ethereumService.defaultAccountAddress)">
+        <pbutton
+          class="${deal.representativeVote(ethereumService.defaultAccountAddress) ? 'acceptButtonActive' : ''}"
+          full-width
+          type="formfield"
+          click.delegate="deal.vote(true, deal.representativeDAO(ethereumService.defaultAccountAddress))"
+        >
           <i class="fas fa-check-circle"></i>
-          Accept
+          ${deal.representativeVote(ethereumService.defaultAccountAddress) === true ? 'Accepted': 'Accept' }
         </pbutton>
-        <pbutton class="declineButtonActive" full-width type="formfield">
+        <pbutton
+          class="${deal.representativeVote(ethereumService.defaultAccountAddress) === false ? 'declineButtonActive' : ''}"
+          full-width
+          type="formfield"
+          click.delegate="deal.vote(false, deal.representativeDAO(ethereumService.defaultAccountAddress))"
+        >
           <i class="fas fa-times-circle"></i>
-          Decline
+          ${deal.representativeVote(ethereumService.defaultAccountAddress) === false ? 'Declined': 'Decline' }
         </pbutton>
       </pbutton-group>
 
-      <div class="negotiate">
-        <pbutton type="tertiary">
+      <div class="negotiate" if.bind="ethereumService.defaultAccountAddress">
+        <pbutton type="tertiary" click.delegate="goToDiscussions()">
           <img src="/static/comment.svg" />
           Negotiate
         </pbutton>
       </div>
 
-      <p class="bodySmallText votesSubtitle representativesVotesSubtitle">Votes (5/10)</p>
-      <deal-representatives-votes dao.bind="deal.primaryDao"></deal-representatives-votes>
-      <deal-representatives-votes dao.bind="deal.partnerDao"></deal-representatives-votes>
+      <p class="bodySmallText votesSubtitle representativesVotesSubtitle">
+        Votes (${deal.submittedVotes.length}/${deal.allRepresentatives.length})
+      </p>
+      <deal-representatives-votes deal.bind="deal" dao.bind="deal.primaryDao" votes.bind="deal.primaryDAOVotes"></deal-representatives-votes>
```

Comment:
nit: 
Can just bind `deal` here? `dao` and `votes` pull from that, and the component has `deal-` in its name --> dependent on `deal`

Reply:
I thought about this but the `dao` and the `votes` attributes need to be passed separately. Passing the `deal` only is not enough as it won't tell us for what dao to display the tokens for. I could add a `type="primary" / type="partner"` attribute but this would introduce more unnecessary "if statements" in the code.

Reply:
is ok for me as-is though it does look odd.  I would probably add the type attribute to make it less questionable.

Reply:
> deal not enough

I see. Let's leave it, and can revisit later (prob won't :p)